homework/index.js

```diff
@@ -29,19 +29,127 @@
     });
     return elem;
   }
-
+  function singlePageApplication(arr) {
+    //header
+    const div = createAndAppend('div', root, {
+      id: 'header-div',
+    });
+    const h1 = createAndAppend('h1', div, {
+      text: 'HYF Repositories',
+      id: 'header',
+    });
+    const selectMenu = createAndAppend('select', div, {
+      id: 'select-menu',
+    });
+    for (let i = 0; i < arr.length; i++) {
+      const option = createAndAppend('option', selectMenu, {
+        text: arr[i].name,
+        value: i,
+      });
+    }
+    //right and left part
+    const containerDiv = createAndAppend('div', root, {
+      id: 'container',
+      class: 'container',
+    });
+    //right part
+    const rightDiv = createAndAppend('div', containerDiv, {
+      id: 'right-div',
+      class: 'contained',
+    });
+    const tHeads = ['Repository: ', 'Description: ', 'Forks: ', 'Update: '];
+    const table = createAndAppend('table', rightDiv, {
+      id: 'tableOfInformation',
+    });
+    for (let i = 0; i < tHeads.length; i++) {
+      var firstTime = ['name', 'description', 'forks', 'updated_at'];
```

Comment:
Why are you using a `var`? Do you know a better way to solve this using `const`?